src/components/_Labels_/DatasetInfo/index.js

```diff
@@ -0,0 +1,80 @@
+import React from "react"
+import { connect } from "react-redux"
+import styles from "./styles.module.css"
+import { fetchDatasetThunk } from "../../../store/datasetStore"
+
+class DatasetInfo extends React.Component {
+  render() {
+    const datasetInfo = this.props.dataset
+    const descriptions = datasetInfo.description || []
+    const compositions = datasetInfo.composition || []
+    const provenances = datasetInfo.provenance || []
+    const collections = datasetInfo.collection || []
+    const managements = datasetInfo.management || []
```

Comment:
Love the initialization, since it leaves room for us to have empty sections for folks who don't fill out all the questions. We should discuss this in the label build meeting, if we want to have empty sections for info not included, or just not show those sections.